src/main/java/com/book/booksstore/model/Book.java

```diff
@@ -0,0 +1,23 @@
+package com.book.booksstore.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
```

Comment:
don't use @Data for entities
```suggestion
@Getter
@Setter
@Table(name...
```

Reply:
done
![image](https://github.com/user-attachments/assets/117230ef-665c-4d5d-bd05-d2ded4b9bb0b)